tensorflow/contrib/android/java/org/tensorflow/contrib/android/TensorFlowInferenceInterface.java

```diff
@@ -156,6 +156,8 @@ public TensorFlowInferenceInterface(InputStream is) {
       if (VERSION.SDK_INT >= 18) {
         Trace.endSection(); // initializeTensorFlow.
       }
+
+      is.close();
```

Comment:
It might be better to invoke `is.close()` in the `finally` clause instead of here. That way any other exceptions raised in this block still result in the `InputStream` being closed.

Reply:
Done!